src/main/java/bio/overture/ego/token/app/AppTokenContext.java

```diff
@@ -36,4 +34,6 @@
   @NonNull
   @JsonProperty("application")
   private Application appInfo;
+
+  private Set<String> Scope;
```

Comment:
Scope should be lower case so `scope`

Reply:
In  `UserTokenContext` it's also capitalized...should they both be lowercase or just the `AppTokenContext`?